main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	gitrepo "github.com/forbole/bdtool/types/repo"
+	"github.com/forbole/bdtool/utils"
+)
+
+var (
+	REPO_URL         = "https://github.com/forbole/big-dipper-2.0-cosmos"
```

Comment:
```suggestion
	REPO_URL         = "https://github.com/forbole/big-dipper-2.0-cosmos"
```
I think it would be better to ask the user to input the organisation name or repo URL before requesting for GitHub personal token instead of hardcoding its value here